capture_gui/plugins/timeplugin.py

```diff
@@ -61,6 +74,56 @@ def _ensure_start(self, value):
     def _ensure_end(self, value):
         self.end.setValue(max(self.end.value(), value))
 
+    def parse_frames(self, string):
```

Comment:
Let's take this method out of the class and turn it into a function so the logic stays separate, especially because it's not using anything from the instance. It's only a simple algorithm.